receiver/scraperhelper/scrapercontroller.go

```diff
@@ -184,7 +191,16 @@ func (sc *controller) startScraping() {
 
 			sc.tickerCh = ticker.C
 		}
+		if sc.initialDelay > 0 {
+			timer := time.NewTimer(sc.collectionInterval)
+			<-timer.C
+			timer.Stop()
```

Comment:
```suggestion
			<-time.After(sc.initialDelay)
```

Reply:
I had thought about doing that but `time.After` explicitly calls out that it leaks the underlying go channel.

I know it will only happen once but I didn't want to give the wrong impression that it should be used more regularly throughout the collector.

Reply:
Are you talking about the fact that the timer is not collected until it fires https://pkg.go.dev/time#After? Or do I miss something else?

Reply:
Yes, though I remembered it differently. Considering I am not waiting on a secondary channel it should be fine.